connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -1576,11 +1854,24 @@ ConnectorOffsetBackingStore offsetStoreForRegularSourceConnector(
         }
     }
 
-    // Visible for testing
+    /**
+     * Builds and returns an offset backing store for an exactly-once source connector. The offset backing store will either be just
+     * the connector-specific offset backing store (if the connector defines a connector-specific offsets topic which happens to be
```

Comment:
The connector doesn't have to define a custom offsets topic, in which case, we also use just a connector-specific store.
```suggestion
     * the connector-specific offset backing store (if the connector's offsets topic is
```